mrpt_bridge/include/mrpt_bridge/utils.h

```diff
@@ -45,14 +45,21 @@ namespace mrpt_bridge {
    *  instead of the package from which macro is actually called.
    */
 	inline void mrptToROSLoggerCallback(const std::string &msg, const mrpt::utils::VerbosityLevel level, const std::string &loggerName, const mrpt::system::TTimeStamp timestamp, void *userParam){
+		
+		// Remove trailing \n if present
+		std::string tmsg = msg;
+		if (tmsg.compare(tmsg.length()-1, tmsg.length(), "\n")==0){
+			tmsg.erase(tmsg.end()-1);
```

Comment:
Please, add a check for empty() before accessing end-1... experience speaks! ;-)